app/build.gradle

```diff
@@ -48,4 +48,15 @@ dependencies {
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
 
     implementation 'androidx.recyclerview:recyclerview:1.2.1'
+
+    implementation "androidx.fragment:fragment-ktx:1.6.0-alpha03"
+    implementation "androidx.lifecycle:lifecycle-viewmodel-ktx:2.5.1"
+
+    implementation 'com.github.bumptech.glide:glide:4.12.0'
+    annotationProcessor 'com.github.bumptech.glide:compiler:4.12.0'
+
+    implementation 'com.squareup.picasso:picasso:2.8'
```

Comment:
Are we using both glide and picasso in our app?

Reply:
I was just testing Picasso, I'll remove this.